fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/FuelPage.tsx

```diff
@@ -442,7 +442,7 @@ export const FuelPage = () => {
                     </div>
 
                     <div
-                        className={`flex justify-center items-center w-20 ${formatRefuelStatusClass()} bg-current`}
+                        className={`flex justify-center items-center w-20  ${formatRefuelStatusClass()} bg-current`}
```

Comment:
Looks unintentional

Reply:
Yep, that was unintentional 😶‍🌫️  Thanks!